docs/comparison_level_composition.md

```diff
@@ -13,7 +13,12 @@ For example, `or_(null_level("first_name"), null_level("surname"))` creates a ch
 
 The Splink comparison level composition functions available for each SQL dialect are as given in this table:
 
-{% include-markdown "./includes/generated_files/comparison_composition_library_dialect_table.md" %}
+||:simple-duckdb: <br> DuckDB|:simple-apachespark: <br> Spark|:simple-amazonaws: <br> Athena|:simple-sqlite: <br> SQLite|:simple-postgresql: <br> PostgreSql|
+|:-:|:-:|:-:|:-:|:-:|:-:|
+|[and_](#splink.comparison_level_composition.and_)|✓|✓|✓|✓|✓|
+|[not_](#splink.comparison_level_composition.not_)|✓|✓|✓|✓|✓|
+|[or_](#splink.comparison_level_composition.or_)|✓|✓|✓|✓|✓|
+
```

Comment:
This table and the other library/dataset ones shouldn't be committed - there has been a more recent fix to the docs build that should mean this substitution happens when you build with `mkdocs` rather than running the script (now deleted) which manually did this substitution

Reply:
Relevant change is [here](https://github.com/moj-analytical-services/splink/pull/1913) - with this version these files won't be changed on-disk, so you shouldn't have to worry about this anymore

Reply:
Good to know, thanks!